tests/contrib/sqlite3/test_sqlite3.py

```diff
@@ -56,70 +56,158 @@ def test_sqlite(self):
+        q = 'select * from sqlite_master'
+        cursor = connection.execute(q)
+        cursor.fetchall()
+
+        spans = tracer.writer.pop()
+
+        eq_(len(spans), 2)
+        
+        execute_span = spans[0]
+        fetchall_span = spans[1]
+        
+        # Execute span
+        eq_(execute_span.name, 'sqlite.query')
+        eq_(execute_span.span_type, 'sql')
+        eq_(execute_span.resource, q)
+        ok_(execute_span.get_tag('sql.query') is None)
+        eq_(execute_span.error, 0)
+        # Fetchall span
+        eq_(fetchall_span.parent_id, None)
+        eq_(fetchall_span.name, 'sqlite.query.fetchall')
+        eq_(fetchall_span.span_type, 'sql')
+        eq_(fetchall_span.resource, q)
+        ok_(fetchall_span.get_tag('sql.query') is None)
+        eq_(fetchall_span.error, 0)
+
+    def test_sqlite_fetchone_is_traced(self):
+        tracer = get_dummy_tracer()
+        connection = self._given_a_traced_connection(tracer)
+        q = 'select * from sqlite_master'
+        cursor = connection.execute(q)
+        cursor.fetchone()
+
+        spans = tracer.writer.pop()
+
+        eq_(len(spans), 2)
+
+        execute_span = spans[0]
+        fetchone_span = spans[1]
+
+        # Execute span
+        eq_(execute_span.name, 'sqlite.query')
+        eq_(execute_span.span_type, 'sql')
+        eq_(execute_span.resource, q)
+        ok_(execute_span.get_tag('sql.query') is None)
+        eq_(execute_span.error, 0)
+        # Fetchone span
+        eq_(fetchone_span.parent_id, None)
+        eq_(fetchone_span.name, 'sqlite.query.fetchone')
+        eq_(fetchone_span.span_type, 'sql')
+        eq_(fetchone_span.resource, q)
+        ok_(fetchone_span.get_tag('sql.query') is None)
+        eq_(fetchone_span.error, 0)
+
+    def test_sqlite_fetchmany_is_traced(self):
+        tracer = get_dummy_tracer()
+        connection = self._given_a_traced_connection(tracer)
+        q = 'select * from sqlite_master'
+        cursor = connection.execute(q)
+        cursor.fetchmany(123)
+
+        spans = tracer.writer.pop()
+
+        eq_(len(spans), 2)
+
+        execute_span = spans[0]
+        fetchmany_span = spans[1]
+
+        # Execute span
+        eq_(execute_span.name, 'sqlite.query')
+        eq_(execute_span.span_type, 'sql')
+        eq_(execute_span.resource, q)
+        ok_(execute_span.get_tag('sql.query') is None)
+        eq_(execute_span.error, 0)
+        # Fetchmany span
+        eq_(fetchmany_span.parent_id, None)
+        eq_(fetchmany_span.name, 'sqlite.query.fetchmany')
+        eq_(fetchmany_span.span_type, 'sql')
+        eq_(fetchmany_span.resource, q)
+        ok_(fetchmany_span.get_tag('sql.query') is None)
+        eq_(fetchmany_span.error, 0)
+        eq_(fetchmany_span.get_tag('db.fetch.size'), '123')
+
     def test_sqlite_ot(self):
-        """Ensure sqlite works with the opentracer."""
+        '''Ensure sqlite works with the opentracer.'''
```

Comment:
probably a replace all typo
```suggestion
        """Ensure sqlite works with the opentracer."""
```